### DISCUSSION TITLE
Fft

### DISCUSSION THREAD

**Discussion by AshtonSBradley** [JuliaSIMD/LoopVectorization.jl#403] [Ideas]
A naive question, but could LoopVectorization be used to speed up fft?

**Comment by chriselrod** [comment]
It's probably possible to speed up an FFT implementation with it.
It'll take reinterpreting to `Float64` arrays for now and it's also probably worth looking into using vfmaddsub and vfmsubadd, e.g. https://github.com/JuliaSIMD/LoopVectorization.jl/blob/0d629549f22595189fae7249a0e203d24e983031/test/shuffleloadstores.jl#L28
but this will be handled automatically in the rewrite.
